src/routes/recovery/recovery.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { Body, Controller, HttpCode, Param, Post } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { AddRecoveryModuleDto } from '@/routes/recovery/entities/add-recovery-module.dto.entity';
+import { RecoveryService } from '@/routes/recovery/recovery.service';
+import { AddRecoveryModuleDtoValidationPipe } from '@/routes/recovery/pipes/add-recovery-module.validation.pipe';
+
+@ApiTags('recovery')
+@Controller({
+  version: '1',
+})
+export class RecoveryController {
+  constructor(private readonly recoveryService: RecoveryService) {}
+
+  @HttpCode(200)
+  @Post('chains/:chainId/safes/:safeAddress/recovery')
```

Comment:
I'd move this as a prefix of the controller as this should be the base path for any recovery feature.

Reply:
I moved it in 193e5cc012a581c84310250f9ebb28e0567a3ced.